packages/integrations/react/src/index.ts

```diff
@@ -94,18 +96,51 @@ export default function ({
 	exclude,
 	babel,
 	experimentalReactChildren,
+	devtools,
 }: ReactIntegrationOptions = {}): AstroIntegration {
+	let config: AstroConfig;
+	let isDev: boolean;
+	let devtoolsProcess: ChildProcess;
+
 	return {
 		name: '@astrojs/react',
 		hooks: {
 			'astro:config:setup': ({ command, addRenderer, updateConfig, injectScript }) => {
+				isDev = command === 'dev';
 				addRenderer(getRenderer());
 				updateConfig({
 					vite: getViteConfiguration({ include, exclude, babel, experimentalReactChildren }),
 				});
-				if (command === 'dev') {
+				if (isDev) {
 					const preamble = FAST_REFRESH_PREAMBLE.replace(`__BASE__`, '/');
 					injectScript('before-hydration', preamble);
+					if (devtools) {
+						injectScript(
+							'head-inline',
+							`
+						const script = document.createElement("script");
+						script.src = "http://localhost:8097";
```

Comment:
Erm, is this an expected port for the devtools chrome extension? Very surprised to see this 😅 Definitely worth a detailed comment if so

Reply:
Yes I believe it's expected for non supported browsers: https://react.dev/learn/react-developer-tools#safari-and-other-browsers. The docs mention having to insert a script tag but since we can't currently with the integration API, I create the script programmatically. Again, I don't mind not shipping this thing at all because it looks hacky to me